src/rpc/blockchain.cpp

```diff
@@ -1824,14 +1829,27 @@ static UniValue getblockstats(const JSONRPCRequest& request)
     std::vector<std::pair<CAmount, int64_t>> feerate_array;
     std::vector<int64_t> txsize_array;
 
+    // The Genesis block and the repeated BIP30 blocks don't change the UTXO
+    // set, so they can be excluded from the statistics
+    const bool is_genesis = pindex->nHeight == 0;
+    const bool is_bip30_repeat = (pindex->nHeight == 91880 && pindex->GetBlockHash() == uint256S("0x00000000000743f190a18c5577a3c2d2a1f610ae9601ac046a38084ccb7cd721")) ||
+                                 (pindex->nHeight == 91842 && pindex->GetBlockHash() == uint256S("0x00000000000a4d0a398161ffc163c503763b1f4360639393e0e4c8e300e0caec"));
```

Comment:
It would be nice to abstract this to a block-aware `IsUnspendable` instead.

Reply:
+1, it would be better not to  repeat this in multiple places

Reply:
Right, but it's a bit more complicated because these blocks' coinbases are not actually unspendable, they are the ones that overwrote the other previous coinbases aifaik. But I have refactored both of these out into helper functions that would also be helpful to me for Coinstatsindex. Let me know what you think about the naming, 'repeat' was the best I came up with.